examples/configFiles/deepMedic/train/trainConfig.cfg

```
@@ -193,7 +195,7 @@ typeOfSamplingForVal = 1
 
 #  +++++ Validation fully segmenting whole volumes +++++
 #  [Optional] How often (epochs) to perform validation by fully inferring validation volumes. Time consuming. Default: 1
-numberOfEpochsBetweenFullInferenceOnValImages = 5
+numberOfEpochsBetweenFullInferenceOnValImages = 1
```

This changes the default trainConfig arbitrarily. Nothing to do with pr change. Avoid such changes, they can go unnoticed.
```suggestion
numberOfEpochsBetweenFullInferenceOnValImages = 5
```